addons/binding/org.openhab.binding.rotelra1x/OSGI-INF/RotelRa1xHandlerFactory.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The OSGI-INF files can be generated with `@Component` This is recommended. In that case also add a .gitignore file to the OSGI-INF directory with content: `*.xml` to not commit the newly generated xml files.